src/ert/simulator/batch_simulator_context.py

```diff
@@ -34,6 +34,35 @@
 Status = namedtuple("Status", "waiting pending running complete failed")
 
 
+class JobStatus(Enum):
```

Comment:
Should we call this "DeprecatedJobStatus" or something, to signify that this is going out?